### PR TITLE
Create Event “2022/07/2022-07-29-2022-federal-plain-language-summit”

### PR DESCRIPTION
**Pull request by Gabr1elleKING**
Automatically generated by Netlify CMS
 
@afeijoo here is the updated preview link: https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/cms/events/2022/07/2022-07-29-2022-federal-plain-language-summit/event/2022/08/24/2022-federal-plain-language-summit/. I removed Rachael from the list of authors for now so we can troubleshoot further, but get the page up.

### CODE REVIEW

File: content/events/2022/07/2022-07-29-2022-federal-plain-language-summit.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+* [Visit PlainLanguage.gov](https://www.plainlanguage.gov/)
+* [Plain Language Summit 2020](https://digital.gov/event/2020/10/27/plain-language-summit-2020/)
+* [2021 Federal Plain Language Summit](https://digital.gov/event/2021/09/21/2021-federal-plain-language-summit/) 
+* [Join the Community of Practice](https://digital.gov/communities/plain-language/)
+* [Follow @GovPlainLang on Twitter](https://twitter.com/govplainlang)
+* [Find additional plain language resources on Digital.gov](https://digital.gov/topics/plain-language/)
+
+- - -
+
+The [Plain Language Action and Information Network (PLAIN)](https://www.plainlanguage.gov/) is a group of federal employees from different agencies and specialties who support the use of clear communication in government writing. [Join the Community of Practice](https://digital.gov/communities/plain-language/).
```
Comment (Gabr1elleKING) [inline content/events/2022/07/2022-07-29-2022-federal-plain-language-summit.md:82]:
```suggestion
_The [Plain Language Action and Information Network (PLAIN)](https://www.plainlanguage.gov/) is a group of federal employees from different agencies and specialties who support the use of clear communication in government writing. [Join the Community of Practice](https://digital.gov/communities/plain-language/)._
```

File: content/events/2022/07/2022-07-29-2022-federal-plain-language-summit.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+* Different ways to run tournament brackets;
+* How to build education into tournament brackets;
+* How to manage situations where colleagues (or a boss!) responds defensively; and
+* Challenges that we've experienced.
+
+An activity like this complements formal plain language training programs. It can be a way to get a wider audience to think about their jargon usage—and, therefore, thinking about plain language throughout their work. It's also pretty fun. Our champions from the last two years are "bleeding edge" and "boil the ocean."
+
+In this session you will hear from the following speaker:
+
+* **Laura Rabuck ** &mdash; Research Health Science Specialist / Lead Content Strategist, United States Department of Veterans Affairs (VA)
```
Comment (Gabr1elleKING) [inline content/events/2022/07/2022-07-29-2022-federal-plain-language-summit.md:119]:
```suggestion
* **Laura Rabuck** &mdash; Research Health Science Specialist / Lead Content Strategist, United States Department of Veterans Affairs (VA)
```

File: content/events/2022/07/2022-07-29-2022-federal-plain-language-summit.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+The presentation highlights how we use awful examples of our agency's own internal memos, emails, and other communication pieces as a powerful plain writing training strategy. Showing lots of “before-and-after” examples of the kinds of pieces we see every day around the agency helps federal employees understand that plain writing isn’t about oversimplifying for a low-literacy public – it’s a research-based science that helps EVERYONE get better results.
+ 
+Participants will learn how we use real examples of communication pieces from around our agency to help federal employees see why plain writing is relevant to their work, even if they don’t ever write for the public. We’ll share how we rewrite and redesign these pieces in a non-threatening, relatable way that helps feds recognize how their current communication approaches (“the way we’ve always done it”) might be sabotaging their results.
+ 
+In this session you will hear from the following speaker:
+
+* **Heather K. Holland ** &mdash; Communications & Usability Specialist, Department of Health and Human Services (HHS) Centers for Medicare & Medicaid Services (CMS)
```
Comment (Gabr1elleKING) [inline content/events/2022/07/2022-07-29-2022-federal-plain-language-summit.md:100]:
```suggestion
* **Heather K. Holland** &mdash; Communications & Usability Specialist, Department of Health and Human Services (HHS) Centers for Medicare & Medicaid Services (CMS)
```

**Review by Gabr1elleKING — COMMENTED** [review]
Formatting updates

File: content/events/2022/07/2022-07-29-2022-federal-plain-language-summit.md (hunk 2 above)
Comment (ToniBonittoGSA) [reply content/events/2022/07/2022-07-29-2022-federal-plain-language-summit.md]: fixed

File: content/events/2022/07/2022-07-29-2022-federal-plain-language-summit.md (hunk 3 above)
Comment (ToniBonittoGSA) [reply content/events/2022/07/2022-07-29-2022-federal-plain-language-summit.md]: fixed

File: content/events/2022/07/2022-07-29-2022-federal-plain-language-summit.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+topics:
+  - content
+  - accessibility
+  - writing
+  - content-strategy
+  - user-experience
+  - plain-language
+  - cx
+# See all authors at https://digital.gov/authors
+authors:
```
Comment (Gabr1elleKING) [inline content/events/2022/07/2022-07-29-2022-federal-plain-language-summit.md:22]: @mejiaj when I add `dr-rachael-bradley-montgomery` here.

**Comment by Gabr1elleKING** [comment]
@mejiaj - so if I add Rachael in the author list now, the circleci fails (at testing - https://app.circleci.com/pipelines/github/GSA/digitalgov.gov/11344/workflows/02458c12-430f-43fb-a9a9-08da7d3e6534/jobs/26214). If I remove and don't include her, the PR goes through just fine. I'm not sure what to do other than submit the PR without her listed until we figure out what happened with the author, thoughts?

**Review by afeijoo — APPROVED** [review]

**Pull request merged**